src/test/java/org/folio/circulation/domain/policy/FixedLoanPolicyRenewalDueDateCalculationTests.java

```diff
@@ -55,10 +91,15 @@ public void shouldFailWhenLoanDateIsAfterOnlyScheduleAvailable() {
 
     DateTime renewalDate = new DateTime(2019, 1, 1, 8, 10, 45, DateTimeZone.UTC);
 
-    final Result<Loan> result = loanPolicy.renew(loan, renewalDate);
+    String requestId = UUID.randomUUID().toString();
+    RequestQueue requestQueue = creteRequestQueue(requestId, RequestType.HOLD);
+
+    final Result<Loan> result = loanPolicy.renew(loan, renewalDate, requestQueue);
 
     assertThat(result, hasValidationFailure(
       EXPECTED_REASON_DATE_FALLS_OUTSIDE_DATE_RANGES));
+
+    assertThat(result, not(hasValidationFailure(EXPECTED_REASON_OPEN_RECALL_REQUEST)));
```

Comment:
Please add assert on errors count instead.
If only one particular error is expected - check for count==1.

Reply:
Added